plugins/sonar-batch-maven-plugin/src/main/java/org/sonar/plugins/batch/maven/MavenProjectBuilder.java

```diff
@@ -51,7 +51,8 @@ public void build(Context context) {
   private void setMavenProjectIfApplicable(ProjectDefinition definition) {
     if (mavenSession != null) {
       String moduleKey = definition.getKey();
-      for (MavenProject mavenModule : (List<MavenProject>) mavenSession.getSortedProjects()) {
+      for (MavenProject mavenModule : (List<MavenProject>) mavenSession.getProjects()) {
```

Comment:
:new: ![MINOR](https://raw.githubusercontent.com/henryju/image-hosting/master/minor.png)Remove this unnecessary cast to "List".[![...](https://raw.githubusercontent.com/henryju/image-hosting/master/more.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1905)